packages/ag-charts-website/src/content/gallery/_examples/customised-waterfall/main.ts

```diff
@@ -0,0 +1,48 @@
+import { AgChartOptions, AgEnterpriseCharts } from 'ag-charts-enterprise';
+import { getData } from './data';
+
+const options: AgChartOptions = {
+    container: document.getElementById('myChart'),
+    data: getData(),
+    title: {
+        text: 'UK Government Budget',
+    },
+    subtitle: {
+        text: 'All values in £ billions',
+    },
+    series: [
+        {
+            type: 'waterfall',
+            xKey: 'financials',
+            xName: 'Financials',
+            yKey: 'amount',
+            yName: 'Amount',
+            item: {
+                positive: {
+                    fill: '#4A90E2',
+                    stroke: '#4A90E2',
+                },
+                negative: {
+                    fill: '#FF6B6B',
+                    stroke: '#FF6B6B',
+                },
+                total: {
+                    name: 'Total / Subtotal',
+                    fill: '#404066',
+                    stroke: '#404066',
+                },
```

Comment:
Shouldn't we avoid having these inline styles?

Reply:
Yes for sure, but this is a customisation example shifted from the docs page for waterfall, it will probably be changed again when we get to making the gallery examples good

Reply:
I can remove the styles for now